plugins/polio/api/perfomance_dashboard/views.py

```
@@ -0,0 +1,98 @@
+    - `country` (ID of the country OrgUnit)
+    - `country_block` (ID of the parent OrgUnit of the country, e.g., a region)
+    - `status` (draft, commented, final)
+    - `antigen` (bOPV, nOPV2, etc.)
+
+    The permissions are structured as follows:
+    - **Read-only**: Can only list and retrieve plans.
+    - **Non-admin**: Can create and update plans.
+    - **Admin**: Can delete plans.
+    """
+
+    # The serializer_class is removed because 'get_serializer_class' is implemented
+    # DRF will call get_serializer_clas() instead of looking for this attribute
+    # serializer_class =
+    pagination_class = PerformanceDashboardPagination
+    filter_backends = [
+        filters.OrderingFilter,
+        django_filters.rest_framework.DjangoFilterBackend,
+    ]
+    filterset_class = PerformanceDashboardFilter
+    ordering_fields = ["date", "country__name", "status", "antigen", "updated_at"]
+    http_method_names = ["get", "post", "patch", "delete"]
+
+    def get_queryset(self):
+        """
+        Get the queryset for the view, filtered for the current user's account.
+        """
+        return (
+            PerformanceDashboard.objects.filter_for_user(self.request.user)
+            .select_related("country", "created_by", "updated_by")
+            .order_by("-date")
+        )
+
+    def get_permissions(self):
+        """
+        Instantiate and return the list of permissions that this view requires,
+        based on the action being performed.
+        """
+        if self.action in ["list", "retrieve"]:
+            permission_classes = [HasPerformanceDashboardReadOnlyPermission]
+        elif self.action in ["create", "partial_update"]:
+            permission_classes = [HasPerformanceDashboardWritePermission]
+        elif self.action == "destroy":
+            permission_classes = [HasPerformanceDashboardAdminPermission]
+        else:
+            permission_classes = [permissions.IsAuthenticated]
```

All scenarios are covered here, if a new one is added and somebody forgets to add it here, let it crash
```suggestion
```